node/src/mpc_client.rs

```diff
@@ -168,12 +168,13 @@ impl MpcClient {
         mut debug_receiver: tokio::sync::broadcast::Receiver<DebugRequest>,
     ) {
         let mut tasks = AutoAbortTaskCollection::new();
-        let mut pending_signatures = PendingRequests::new(
-            Clock::real(),
-            self.client.all_participant_ids(),
-            self.client.my_participant_id(),
-            self.client.clone(),
-        );
+        let mut pending_signatures =
+            PendingRequests::<SignatureRequest, ChainSignatureRespondArgs>::new(
```

Comment:
this is not necessary, but I though it would be more readable making types explicit